alby/alby_oauth_service.go

```diff
@@ -645,11 +646,38 @@ func (svc *albyOAuthService) ConsumeEvent(ctx context.Context, event *events.Eve
 		return
 	}
 
-	// TODO: we should have a whitelist rather than a blacklist, so new events are not automatically sent
+	// ensure we do not send unintended events to Alby API
+	eventWhitelist := []string{
+		"nwc_backup_channels",
+		"nwc_payment_received",
+		"nwc_payment_sent",
+		"nwc_payment_failed",
+		"nwc_app_created",
+		"nwc_app_deleted",
+		"nwc_unlocked",
+		"nwc_node_sync_failed",
+		"nwc_outgoing_liquidity_required",
+		"nwc_incoming_liquidity_required",
+		"nwc_budget_warning",
+		"nwc_channel_ready",
+		"nwc_channel_closed",
+		"nwc_permission_denied",
+		"nwc_started",
+		"nwc_stopped",
+		"nwc_node_started",
+		"nwc_node_start_failed",
+		"nwc_node_stop_failed",
+		"nwc_node_stopped",
+	}
+
+	if !slices.Contains(eventWhitelist, event.Event) {
+		logger.Logger.WithField("event", event).Debug("Skipped sending non-whitelisted event to alby events API")
+		return
+	}
 
 	// TODO: rename this config option to be specific to the alby API
 	if !svc.cfg.GetEnv().LogEvents {
-		logger.Logger.WithField("event", event).Debug("Skipped sending to alby events API")
+		logger.Logger.WithField("event", event).Debug("Skipped sending to alby events API (alby event logging disabled)")
```

Comment:
nit: first check if it's enabled and if it's enabled we do the whitelist check?

Reply:
It's just a bit harder to test without enabling events in dev mode. But I will switch it